async/build.gradle.kts

```diff
@@ -1,18 +1,15 @@
-import dependencies.Dependencies
-import dependencies.Versions
-
 plugins {
-    id("com.android.library")
-    id("kotlin-android")
+    alias(libs.plugins.androidLibrary)
+    alias(libs.plugins.kotlinAndroid)
 }
 
 apply(from = "../config/quality.gradle")
 
 android {
-    compileSdk = Versions.android_compile_sdk
+    compileSdk = libs.versions.compileSdk.get().toInt()
```

Comment:
I'm thinking it might make sense to still declare the compile/target/mid SDKs in as constants in Gradle (example below). It doesn't seem like version catalog lets us define integer values. They're also more like shared configuration than library versions.

```kotlin
buildscript {
    ext {
        minSdkVersion = 21
        targetSdkVersion = 34
        compileSdkVersion = 34
    }
...
```

Reply:
In kotlin DSL this would be like this:
```
    compileSdk = rootProject.extra["compileSdkVersion"] as Int
```

so it would also require casting to Int. Given that I don't think this approach would be any better.

Reply:
Oh, wow. Kotlin makes that much worse - in Groovy it'd be `rootProject.ext.minSdkVersion`. Let's leave it as-is then.